index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/styles.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header class="container">
+       <h1 class="name">John Doe</h1>
+        <nav class="hamburger"> 
+            <div></div>
+            <div></div>
+            <div></div>
+        </nav>
+    </header>
+<section class="headline">
+    <h2 class="title">Hey There.
+        I'm Alex</h2>
+    <h3 class="subtitle">I'm a Software Developer</h3>
+    <p class="description">I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don't hestiate to contact me.</p>
+    </section>
+<ul class="icons">
+    <li><img src="icons/Github.png" alt="Github icon"></li>
+    <li><img src="icons/Linkedin.png" alt="LinkedIn icon"></li>
+    <li><img src="icons/Angelist.png" alt="AngelList icon"></li>
+    <li><img src="icons/Twitter.png" alt="Twitter icon"></li>
+    <li><img src="icons/Medium.png" alt="Medium icon"></li>
```

Comment:
Good job for using `<ul>` `<li>` tags for displaying a list of items. As these are links which going to use to take users to another site that's why Please use the `<a>` tag as well.
Example:
```
<ul>
  <li><a href="link_to_your_github_accound"><img src="icons/Github.png" alt="Github icon"><a/></li>
  ...
  ...
  ...
</ul>
```